runner.rb

```diff
@@ -7,6 +7,10 @@
 q = Query.new
 r = Response.new 
 
+url = ARGV[0]
+
+puts "Fetching ip address for #{url}, BOI!!!!!"
```

Comment:
```suggestion
puts "Fetching ip address for #{url}"
```

juvenile, unacceptable